.travis.yml

```diff
@@ -94,6 +95,6 @@ jobs:
     # Last stable with highest supported PHP version.
     - php: 8.2
       env: MOODLE_BRANCH=MOODLE_402_STABLE
-    # And oldest stable supported (with lowest supported PHP version).
+    # And older stable supported (with lowest supported PHP version).
     - php: 7.4
-      env: MOODLE_BRANCH=MOODLE_38_STABLE
+      env: MOODLE_BRANCH=MOODLE_39_STABLE
```

Comment:
Same comment here than in the GH workflow self-tests. Maybe we can add that extra setting to decide...

Reply:
Answered above.